contracts/staking/contracts/src/stake/MixinStake.sol

```diff
@@ -42,10 +42,10 @@ contract MixinStake is
         getZrxVault().depositFrom(staker, amount);
 
         // mint stake
-        _incrementCurrentAndNextBalance(_activeStakeByOwner[staker], amount);
+        _increaseCurrentAndNextBalance(_activeStakeByOwner[staker], amount);
```

Comment:
Nice! `increase` and `decrease` are definitely more accurate than `increment` and `decrement`.